src/data/BackendInterface.cpp

```diff
@@ -48,6 +48,12 @@ namespace {
 util::Logger gLog{"Backend"};
 }  // namespace
 
+/**
+ * @brief This namespace implements the data access layer and related components.
+ *
+ * The data layer is responsible for fetching and storing data from the database.
+ * Cassandra and ScyllaDB are currently supported via the `CassandraBackend` implementation.
+ */
```

Comment:
Do we need to do it once somewhere?

Reply:
Yes... :rage4: It needs to be somewhere on top of the namespace, otherwise the namespace does not even appear in the `namespaces` page 👎